src/modengine/settings_loader.cpp

```diff
@@ -10,10 +10,10 @@ using namespace spdlog;
 bool SettingsLoader::load_toml_into(Settings& settings, const fs::path& path)
 {
     try {
-        auto config = toml::parse_file(path.wstring());
+        auto config = toml::parse_file(path.string());
         settings.m_config = config;
     } catch (const toml::parse_error& e) {
-        error("Failed to load config {}", e.what());
+        error("Failed to load config: {}", e.what());
```

Comment:
Making this `"Failed to load config (from '{}'): {}", path.string(), e.what()` would be good.